libcfnet/net.c

```diff
@@ -123,7 +123,7 @@ int SendTransaction(const ConnectionInfo *conn_info,
  *  @return 0 in case of socket closed, -1 in case of other error, or
  *          >0 the number of bytes read.
  */
-int ReceiveTransaction(const ConnectionInfo *conn_info, char *buffer, int *more)
+int ReceiveTransaction(ConnectionInfo *conn_info, char *buffer, int *more)
```

Comment:
We can experiment with adding new arguments or something. In fact ConnectionInfo is modified in lot of places and almost only in this one it was const. This was quite convenient way to modify the ConnectionInfo itself as we are having access to it from all parts of code and it is easy to set/get flag needed.